INSTALL.rst

```diff
@@ -199,8 +199,8 @@ venv
   source ~/envs/skimage-dev/bin/activate
 
   # Install development dependencies
-  pip install -r requirements.txt
-  pip install -r requirements/build.txt
+  pip install -U pip
+  pip install --group build --group dev --group docs --group test
```

Comment:
I was wondering how widely supported this is among package managers. It seems that pip added support for this only ~30 days ago in [version 25.1](https://discuss.python.org/t/announcement-pip-25-1-release/89702)? Should we hold off on this for a bit or do you imagine that most contributors will be able to easily use the most recent pip?

Though, happy to get rid of requirements.txt sooner rather than later.

Reply:
This is not urgent, so we can wait.